src/cosmos/tx/handlers/trigger.rs

```diff
@@ -0,0 +1,115 @@
+                        .map(|msg| msg.to_any().unwrap())
+                        .collect::<Vec<_>>(),
+                    message.memo,
+                    0u16,
+                );
+
+                let sign_tx = res
+                    .map_err(Error::from)
+                    .and_then(|value| value.map_err(Error::from))
+                    .and_then(|account| act.sign_tx(&body, account, act.fee.clone()));
+                (sign_tx, grpc_client)
+            })
+            .then(|(sign_tx, grpc_client), act, _| {
+                async move {
+                    let result: Result<BroadcastTxResult, Error> = match sign_tx {
+                        Ok(tx_bytes) => {
+                            info!("🔥 Broadcast transaction");
+                            grpc_client
+                                .send(BroadcastTx { tx: tx_bytes })
+                                .await
+                                .map_err(Error::from)
+                        }
+                        Err(why) => {
+                            error!("❌ Failed sign transaction: {}", why);
+                            Err(why)
+                        }
+                    };
+                    result
+                }
+                .into_actor(act)
+            })
+            .map(move |tx_result, act, _| {
+                match tx_result.and_then(|i| i.map_err(Error::from)) {
+                    Ok(tx_response) => {
+                        info!(
+                            "Transaction successfully broadcasted : {}",
+                            tx_response.txhash
+                        );
+                        match act.channel_id {
+                            Some(channel_id) => discord_client.do_send(SendMessage {
+                                title: String::from("🚀 Transaction broadcasted!"),
+                                description: format!(
+                                    "\t- 🤝 Transaction hash: {}
+                            \t- ⚙️ Result code : {}
+                            \t- ⛽️ Gas used: {}",
+                                    tx_response.txhash, tx_response.code, tx_response.gas_used
+                                ),
+                                content: {
+                                    let mut str = String::new();
+                                    for sub in subscribers {
+                                        str.push_str(
+                                            &format_args!("{member} ", member = &sub.mention())
+                                                .to_string(),
+                                        );
+                                    }
+                                    str
+                                },
+                                channel_id,
+                            }),
+                            None => {}
+                        };
+                    }
+                    Err(why) => error!("❌ Failed sign transaction {}", why),
```

Comment:
We could add message to say we're sorry ^^

```suggestion
                       Err(why) => {
                        error!("❌ Failed sign transaction {}", why);
                        match act.channel_id {
                            Some(channel_id) => discord_client.do_send(SendMessage {
                                title: String::from("🤷 So sorry, something went wrong"),
                                description: String::from(""),
                                content: {
                                    let mut str = String::new();
                                    for sub in subscribers {
                                        str.push_str(
                                            &format_args!("{member} ", member = &sub.mention())
                                                .to_string(),
                                        );
                                    }
                                    str
                                },
                                channel_id,
                            }),
                            None => {}
                        };
                    }
```

Reply:
Good call ! :relaxed: